platformio.ini

```diff
@@ -637,12 +639,14 @@ lib_ignore  = LiquidCrystal, LiquidTWI2, Adafruit NeoPixel, TMCStepper, U8glib-H
 #
 [env:STM32F070RB_malyan]
 platform    = ststm32
+platform_packages = framework-arduinoststm32@${common.arduinoststm32_ver}
 board       = malyanM200v2
 build_flags = -DSTM32F0xx -DUSBCON -DUSBD_VID=0x0483 '-DUSB_MANUFACTURER="Unknown"' '-DUSB_PRODUCT="ARMED_V1"' -DUSBD_USE_CDC -DHAL_PCD_MODULE_ENABLED
   -O2 -ffreestanding -fsigned-char -fno-move-loop-invariants -fno-strict-aliasing -std=gnu11 -std=gnu++11
+  -DCUSTOM_STARTUP_FILE
   -IMarlin/src/HAL/STM32
 src_filter  = ${common.default_src_filter} +<src/HAL/STM32>
-lib_ignore  = LiquidCrystal, LiquidTWI2, Adafruit NeoPixel, TMCStepper, U8glib-HAL
+lib_ignore  = LiquidCrystal, LiquidTWI2, Adafruit NeoPixel, TMCStepper, U8glib-HAL, SoftwareSerial
 
 #
 # Malyan M300 (STM32F070CB)
```

Comment:
Malyan M300 is not using the version argument, because the variant is uses is only available in 4.10800.
I suspect the M300 isn't currently working due to the timer issues I've seen with it on the SKR Pro, but I'm out of the M300 loop.

Once other environments are able to update to the 4.10800 framework the M300 should be updated to use the same variable as other environments.